core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -118,8 +118,8 @@ public int minWordLength() {
         return minWordLength;
     }
 
-    public void minQueryLength(int minQueryLength) {
-        this.minWordLength = minQueryLength;
+    public void minWordLength(int minWordLength) {
```

Comment:
I'm not sure how much user-facing this API is, maybe also take a note that it has changes in `docs/reference/migration/migrate_3_0.asciidoc`